DemoApp-Kotlin/app/src/main/res/layout/activity_programmatic_banner_ad.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.applovin.enterprise.apps.demoapp.ads.ProgrammaticBannerAdActivity">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
nit: you can get rid of this closing tag (similar to [`content_home.xml`](https://github.com/AppLovin/AppLovin-MAX-SDK-Android/blob/6631ca6591bfafc6e23adda8a3a4126a30a88652/DemoApp-Kotlin/app/src/main/res/layout/content_home.xml))